app.yaml

```diff
@@ -1,8 +1,29 @@
-runtime: nodejs16
-instance_class: F4
+# Node.js 18 is EOL on April 30, 2025
+# https://github.com/nodejs/release#release-schedule
+# Note: Library currently does not run under Node.js 20, will need to pull
+#       changes from upstream closer to v18 EOL
+runtime: nodejs18
+
+# Capability table: https://cloud.google.com/appengine/docs/standard#instance_classes
+# Cost table: https://cloud.google.com/appengine/pricing (go to us-west4)
```

Comment:
I really appreciate all the comments you added, but I especially love it when people include URLs to the source of the relevant information like you did here.

Reply:
It's as much for me as for anyone else! I'm notoriously sieve-minded if I don't have solid notes in place to reproduce something I figured out months ago. Bites me at work sometimes.